Dynamic distance

NoodleSoup0 opened the pull request:
Two Changes:
Added dynamic distance + time to each card
![dynamic-distance](https://github.com/user-attachments/assets/a387f819-a547-4e7f-8f2d-0a652252d2e6)
Saved meetup location from request form:
![image](https://github.com/user-attachments/assets/edd0fe9e-4aa2-44e2-be1a-502b7dab4bba)

Commit:
stuff

Commit:
stuff

Commit:
stuff

Commit:
dynamic_distance

Commit:
issue with styling data

Commit:
Save meetup location to DB and calculate time to cover distance

Commit:
Merge branch 'main' of https://github.com/392-f24/YellowTeam-GoodNeighbor into dynamic-distance

dwheal01 commented on data/mockupdata.json:
Do we need to keep these?

dwheal01 commented on src/components/pages/RequestForm.jsx:
```diff
@@ -11,15 +11,7 @@ import { useState, useEffect } from 'react';
   const location = useLocation();
-  const navigate = useNavigate(); 
-
-  useEffect(() => {
```
did you mean to delete this?

dwheal01 commented on src/components/AuthButton.jsx:
I think this file got delted/moved

dwheal01 commented on src/SignIn.jsx:
this got moved

NoodleSoup0 committed:
Update RequestForm.jsx

Accidentally deleted useEffect from RequestFormPage as pointed out by Diana; restored it
